.npmrc

```diff
@@ -0,0 +1,2 @@
+strict-peer-dependencies=false
+shamefully-hoist=true
```

Comment:
lerna used to also hoist, think its due to the copying we do of our packages — ideally in a future moment we won't need to do this.